client/voucher.py

```diff
@@ -23,7 +23,7 @@ def main(issuer, cid, network, n):
             voucher_uri = f"//{batch_token}/{voucher_token}"
             f.write(voucher_uri)
             print(f"generating voucher {i}: {voucher_uri}")
-            payload = f"encointer-voucher\nV1.0\n{voucher_uri}\n{cid}@{network}\n\n{issuer}"
+            payload = f"encointer-voucher\nV1.0\n{voucher_uri}\n{cid}\n{network}\n{issuer}"
```

Comment:
shouldn't the version read 2.0 now?

Reply:
Ah yes, but as the voucher does only exist in v2.0, it is the only one that does not check the version. If it can decode the field into the `enum` with values v1.0 and v2.0 it treats it as a success.